test/runtests.jl

```diff
@@ -2,15 +2,26 @@ using Zygote, Test
 using Zygote: gradient
 using CUDA: has_cuda
 
+if has_cuda()
```

Comment:
I moved these tests up, so that CI would fail faster on them. They can go back to being last if anyone feels strongly about it. These are the only optional tests, so getting early feedback about whether they are going to be run or not, locally, doesn't seem like a terrible idea.